docs/proposals/odo-v3-cli/odo-list.md

```diff
@@ -0,0 +1,29 @@
+# `odo list`
+
+list all odo related information from the current namespace
+
+## Flags
+- `--namespace` - list components from the given namespace instead of current namespace.
+
+## example
+```
+$ odo list
+Components in the "mynamspace" namespace:
+
+  NAME              TYPE         MANAGED BY       RUNNING IN
+* frontend          nodejs       odo              Dev,Deploy
+  backend           springboot   odo              Deploy
+  created-by-odc    python       Unknown          Unknown
```

Comment:
I think that `TYPE` here should be component, endpoint, service, etc. because `odo list`, as per the discussion we had on call during the past couple of days, is supposed to list all the kinds of resource.